src/projects/yaq/src/public/js/events.js

```diff
@@ -47,3 +47,18 @@ export function initializeButtonEvents(
     });
   }
 }
+
+export function getGreeting() {
+  const currentHour = new Date().getHours();
+
+  if (currentHour < 12) {
+    return "GOOD MORNING";
```

Comment:
instead off manually using uppercase, its better to use function for transform text to uppercase.

Reply:
Few things can be hardcode